### PR TITLE
:bug:fix(lld): fix export to csv account name

### PR DESCRIPTION
**Pull request by LucasWerey**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - export to csv

### 📝 Description

Accounts names were not exported in the csv it's now fixed. 

image

### ❓ Context

- **JIRA or GitHub link**:  [LIVE-14076](https://ledgerhq.atlassian.net/browse/LIVE-14076)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-14076]: https://ledgerhq.atlassian.net/browse/LIVE-14076?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/ledger-live-common/src/csvExport.ts [hunk 1]
```diff
@@ -65,7 +67,10 @@ const fields: Field[] = [
   {
     title: "Account Name",
     // FIXME: we need to inject wallet state if we want the actual user's account name
-    cell: (account, parentAccount) => getDefaultAccountName(getMainAccount(account, parentAccount)),
+    cell: (account, parentAccount, _op, _counterValueCurrency, _countervalueState, walletState) =>
```
Comment (mcayuelas-ledger) [inline libs/ledger-live-common/src/csvExport.ts:70]: why do we need this 3 ? `_op, _counterValueCurrency, _countervalueState`

**Review by mcayuelas-ledger — CHANGES_REQUESTED** [review]

File: libs/ledger-live-common/src/csvExport.ts (hunk 1 above)
Comment (LucasWerey) [reply libs/ledger-live-common/src/csvExport.ts:70]: Because how it's done today each param if walletState is at the third place it would be the operation

Comment (mcayuelas-ledger) [reply libs/ledger-live-common/src/csvExport.ts:69]: indeed ^^

**Review by mcayuelas-ledger — APPROVED** [review]

**Review by themooneer — APPROVED** [review]

**Comment by martin-cech** [comment]
Hello, I suspect this caused a change in CSV export that may not be desired.

When I receive USDT on Arbitrum network, the Account Name in export is now "Tether USD". This is not an account name and I have no idea where it came from. Previously the Account Name for this transaction was "Arbitrum 1". This seems more like a bug to me.

Similarly, for USDC on Arbitrum network the Account Name is "USD Coin" (previously also "Arbitrum 1") and for Abitrum itself the Account Name is just "Arbitrum" (previously "Arbitrum 1").

Can you please help me understand this?

**Pull request merged**